### PR TITLE
address vision with covalent api and multichain tab layout 

### PR DESCRIPTION
**Pull request by Naim-Bijapure**
## Description

multichain layout with tabs. 
currently  using covalent API to fetch the address data. 
image

### CODE REVIEW

**Comment by ZakGriffith** [comment]
Looks like a few linting errors are causing the Vercel build to fail

**Comment by Naim-Bijapure** [comment]
@ZakGriffith  fixed this can you add NEXT_PUBLIC_COVALENT_TOKEN env variable with covalent key in vercel / nextjs settings to work it properly.

**Comment by rin-st** [comment]
@ZakGriffith is it okay that it's pr to main?

File: packages/nextjs/generated/deployedContracts.ts
Comment (rin-st) [inline packages/nextjs/generated/deployedContracts.ts:1]: it seems you don't need this file :)

**Comment by rin-st** [comment]
Looks nice! Hopefully, I'll review and play with it next days

**Comment by ZakGriffith** [comment]
Merged this to a new branch Naim-dev

**Comment by ZakGriffith** [comment]
ENV variable added and it's deployed to https://address-vision-git-naim-dev-buidlguidldao.vercel.app/

File: packages/nextjs/generated/deployedContracts.ts
Comment (Naim-Bijapure) [reply packages/nextjs/generated/deployedContracts.ts:1]: yah wont needed :)

**Pull request merged**